### PR TITLE
Hungarian Translation of the SkyKings Bot

### PR DESCRIPTION
**Pull request by Bilovisnake**
Adds translations for `hu-HU`

- [x] I have followed the guidelines listed in the README.

### CODE REVIEW

File: translations/hu-HU.json [hunk 1]
```diff
@@ -0,0 +1,2970 @@
...
+        "Globális esemény csatornája: {global_event_channel}\n",
+        "SkyBlock dátum/idő csatornája: {skyblock_time}\n",
+        "SkyBlock események: {skyblock_events}\n",
+        "Műveleti napló: {scammer_logs}\n",
+        "Scammer művelet: {scammer_action}\n",
+        "IRL Kereskedő művelet: {trader_action}\n",
+        "Makrózó művelet: {macroer_action}\n",
+        "Rattelő művelet: {phisher_action}\n",
+        "\n",
+        "Módosítani tudod a konfigurációkat a '/config edit' paranccsal!"
```
Comment (plun1331) [inline translations/hu-HU.json]: the command name here should be translated and placed in an inline codeblock (\`\`)

File: translations/hu-HU.json [hunk 2]
```diff
@@ -0,0 +1,2970 @@
...
+        "VIP: {vip}\n",
+        "VIP+: {vip_plus}\n",
+        "MVP: {mvp}\n",
+        "MVP+: {mvp_plus}\n",
+        "MVP++: {mvp_plus_plus}\n",
+        "YouTube Rang: {youtube}\n",
+        "Hypixel Dolgozó: {staff}\n",
+        "Becenév formátum: {nickname}\n",
+        "\n",
+        "Módosítani tudod a konfigurációkat a '/config edit' paranccsal!"
```
Comment (plun1331) [inline translations/hu-HU.json]: Same as above

File: translations/hu-HU.json [hunk 3]
```diff
@@ -0,0 +1,2970 @@
...
+      },
+      {
+        "name": "értesítés",
+        "description": "Az új opció értéke. Hagyd üresen, ha alaphelyzetbe szeretnéd állítani."
+      }
+    ],
+    "responses": {
+      "premiumOnly": [
+        "Ez az opció csak a prémium szervereken használható.\n",
+        "Ha többet szeretnél tudni, használd a '/premium info' parancsot."
```
Comment (plun1331) [inline translations/hu-HU.json]: Same as above

File: translations/hu-HU.json [hunk 4]
```diff
@@ -0,0 +1,2970 @@
...
+      ],
+      "editedSuccessfully": [
+        "{option} {value}-re lett állítva!!"
```
Comment (plun1331) [inline translations/hu-HU.json]: Double exclamation mark

```suggestion
        "{option} {value}-re lett állítva!"
```

File: translations/hu-HU.json [hunk 5]
```diff
@@ -0,0 +1,2970 @@
...
+      },
+      {
+        "name": "role",
+        "description": "A role ami a ranghoz lesz kötve. Hagyd üresen, ha nem akarsz role-t."
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincs ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a  `/config guilds add-guild` paranccsal."
```
Comment (plun1331) [inline translations/hu-HU.json]: Same as first comment

File: translations/hu-HU.json [hunk 6]
```diff
@@ -0,0 +1,2970 @@
...
+      },
+      {
+        "name": "role",
+        "description": "A role, ami a ranghoz van kötve. Hagyd üresen, ha nem akarsz role-t."
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincs ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a `/config guilds add-guild` paranccsal."
```
Comment (plun1331) [inline translations/hu-HU.json]: Same as above

File: translations/hu-HU.json [hunk 7]
```diff
@@ -0,0 +1,2970 @@
...
+      },
+      {
+        "name": "rang",
+        "description": "A törölnivaló rang."
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincs ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a `/config guilds add-guild` paranccsal."
```
Comment (plun1331) [inline translations/hu-HU.json]: Same as above

File: translations/hu-HU.json [hunk 8]
```diff
@@ -0,0 +1,2970 @@
...
+      },
+      {
+        "name": "érték",
+        "description": "A követelmény értéke. Írj be akármilyen számot a 'API beállítások bekapcsolva' követelményhez."
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincs ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a `/config guilds add-guild` paranccsal."
```
Comment (plun1331) [inline translations/hu-HU.json]: Same as above

File: translations/hu-HU.json [hunk 9]
```diff
@@ -0,0 +1,2970 @@
...
+          "Fairy Souls",
+          "Súlyozás",
+          "API beállítások bekapcsolva"
+        ]
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincs ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a `/config guilds add-guild` paranccsal."
```
Comment (plun1331) [inline translations/hu-HU.json]: ame as above

File: translations/hu-HU.json [hunk 10]
```diff
@@ -0,0 +1,2970 @@
...
+      },
+      {
+        "name": "érték",
+        "description": "A követelmény új értéke."
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincs ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a `/config guilds add-guild` paranccsal."
```
Comment (plun1331) [inline translations/hu-HU.json]: Same as above
I give up on marking all of these

File: translations/hu-HU.json [hunk 11]
```diff
@@ -0,0 +1,2970 @@
...
+          "Fear Mongerer",
+          "Dark Auction",
+          "Mayor Election kezdődik",
+          "Bank Interest"
+        ]
+      }
+    ],
+    "responses": {
+      "enabled": [" **{event}** bekapcsolva."],
+      "disabled": [" **{event}** kikapcsolva."]
```
Comment (plun1331) [inline translations/hu-HU.json]: Spacing, unless this was also done in the English translation
```suggestion
      "enabled": ["**{event}** bekapcsolva."],
      "disabled": ["**{event}** kikapcsolva."]
```

File: translations/hu-HU.json [hunk 12]
```diff
@@ -0,0 +1,2970 @@
...
+        "Tag: {tag}\n",
+        "XP: {xp:,}\n",
+        "Level: {level:,}\n",
+        "Preferred Games: {prefgames}"
+      ]
+    }
+  },
+  "guild members": {
+    "name": "tagok",
+    "description": "Kijelez egy  listát a guild tagjairól.",
```
Comment (plun1331) [inline translations/hu-HU.json]: Spacing
```suggestion
    "description": "Kijelez egy listát a guild tagjairól.",
```

File: translations/hu-HU.json [hunk 13]
```diff
@@ -0,0 +1,2970 @@
...
+      "GIFTS_GIVEN_LABEL": [
+        "Gift-ek"
+      ],
+      "startMessage": [
+        "{created_by.mention} elindított egy {gvg}{event} eseményt! Itt van, hogy működik ez:\n",
+        "1. Csatlakozol az eseménybe,\n",
+        "2. A bot elkezdi követni hogy mennyi XP-d van,\n",
+        "3. És el leszel helyezve a ranglistán az XP-d szerint.\n",
+        "Ez az üzenet fog ranglistaként szolgálni, és frissülni fog minden {when}-ben. Sok szerencsét és jó szórakozást!\n",
+        "**This event ends on {end_date} ({end_relative})**"
```
Comment (plun1331) [inline translations/hu-HU.json]: Translation where

File: translations/hu-HU.json [hunk 14]
```diff
@@ -0,0 +1,2970 @@
...
+      "joined": [
+        "Sikeresen csatlakoztál az eseményhez!"
+      ],
+      "notJoined": [
+        "Nem vagy az eseményben."
+      ],
+      "position": [
+        "Jelenleg {xp:,} {label}-d van, és ezzel a #{pos:,}-dik vagy a {players:,} játékosból."
+      ],
+      "jsonify": ["Dump to JSON File"]
```
Comment (plun1331) [inline translations/hu-HU.json]: Does this have a translation

**Review by plun1331 — CHANGES_REQUESTED** [review]
Have a few more checks to run once I get home after the 28th

**Comment by Bilovisnake** [comment]
I think I fixed it, but I don't know if Github updated it.

**Comment by Altpapier** [comment]
Pretty sure the name should just be hu instead of hu-HU

**Comment by plun1331** [comment]
Added extra checks, please fix the following issues:
```
Incorrect list length at hu/config view/responses/description.1
Missing keys in hu/config guilds view/responses: noGuilds
Missing keys in hu/config guilds add-guild/responses: maxGuilds, premiumRequired
Missing keys in hu/lookup user/responses: TRADER, TRADER_NAME
Extra keys in hu/lookup user/responses: IRL_TRADER_NAME, IRL_TRADER
Missing keys in hu/verify link/responses: cancelled
```

File: translations/hu.json [hunk 15]
```diff
@@ -0,0 +1,2983 @@
...
+    "description": "Tekintsd meg a szervered konfigurációját.",
+    "options": [],
+    "responses": {
+      "title.1": [
+        "{ctx.guild.name} konfigurációja"
+      ],
+      "description.1": [
+        "Automatikus válasz bekapcsolása: {autoreply}\n",
+        "Hypixel Status Ping: {status_ping}\n",
+        "Hypixel Status Channel: {status_channel}\n",
```
Comment (plun1331) [inline translations/hu.json]: I've just added these in, please translate

File: translations/hu.json [hunk 16]
```diff
@@ -0,0 +1,2983 @@
...
+    "responses": {
+      "neverJoined": [
+        "**{name}** sose csatlakozott a Hypixel-re."
+      ],
+      "title": ["{name}'s Státusza"],
+      "offline": ["**{name}** jelenleg offline."],
+      "noData": ["Nem voltam képes **{name}** státuszát lehozni."],
+      "onlineCantLocate": ["**{name}** jelenleg online, de a státuszuk kikapcsolt API beállítást jeleznek."],
+      "online": ["**{name}** jelenleg online. Viszont nem tudom, hogy hol van."],
+      "onlineWithMode": ["**{name}** jelenleg {game}-t játszik.\nMód: {mode}."],
```
Comment (plun1331) [inline translations/hu.json:1530]: Slide this in here as well
https://github.com/SkyKings-Guild/translations/blob/52c3263324591342a900b9b8ace1f073fc2ee300/translations/en-US.json#L1531

**Review by plun1331 — CHANGES_REQUESTED** [review]

File: translations/hu.json [hunk 17]
```diff
@@ -0,0 +1,2984 @@
...
+        "Globális esemény csatornája: {global_event_channel}\n",
+        "SkyBlock dátum/idő csatornája: {skyblock_time}\n",
+        "SkyBlock események: {skyblock_events}\n",
+        "Műveleti napló: {scammer_logs}\n",
+        "Scammer művelet: {scammer_action}\n",
+        "IRL Kereskedő művelet: {trader_action}\n",
+        "Makrózó művelet: {macroer_action}\n",
+        "Rattelő művelet: {phisher_action}\n",
+        "\n",
+        "Módosítani tudod a konfigurációkat a '/config módosítás' paranccsal!"
```
Comment (plun1331) [inline translations/hu.json]: The whole command name should be translated, including top level
Should also be in backticks for `this effect`

File: translations/hu.json [hunk 18]
```diff
@@ -0,0 +1,2984 @@
...
+        "VIP: {vip}\n",
+        "VIP+: {vip_plus}\n",
+        "MVP: {mvp}\n",
+        "MVP+: {mvp_plus}\n",
+        "MVP++: {mvp_plus_plus}\n",
+        "YouTube Rang: {youtube}\n",
+        "Hypixel Dolgozó: {staff}\n",
+        "Becenév formátum: {nickname}\n",
+        "\n",
+        "Módosítani tudod a konfigurációkat a '/config módosítás' paranccsal!"
```
Comment (plun1331) [inline translations/hu.json]: The whole command name should be translated, including top level
Should also be in backticks for `this effect`

File: translations/hu.json [hunk 19]
```diff
@@ -0,0 +1,2984 @@
...
+      },
+      {
+        "name": "értesítés",
+        "description": "Az új opció értéke. Hagyd üresen, ha alaphelyzetbe szeretnéd állítani."
+      }
+    ],
+    "responses": {
+      "premiumOnly": [
+        "Ez az opció csak a prémium szervereken használható.\n",
+        "Ha többet szeretnél tudni, használd a '/premium infomáció' parancsot."
```
Comment (plun1331) [inline translations/hu.json]: The whole command name should be translated, including top level
Should also be in backticks for `this effect`

File: translations/hu.json [hunk 20]
```diff
@@ -0,0 +1,2984 @@
...
+    "responses": {
+      "addedGuild": [
+        "A **{guild}** nevű guild hozzá lett adva a szerverhez!"
+      ],
+      "guildNotFound": [
+        "A **{guild}** nevű guild nem létezik!"
+      ],
+      "premiumRequired": [
+        "Prémium szükséges ahhoz, hogy háromnál több guild-et adj hozzá a szerveredhez.\n",
+        "Több információért futtasd a /premium infó parancsot."
```
Comment (plun1331) [inline translations/hu.json]: The whole command name should be translated, including top level
Should also be in backticks for `this effect`

File: translations/hu.json [hunk 21]
```diff
@@ -0,0 +1,2984 @@
...
+      "responses": {
+        "guildNotFound": [
+          "A **{guild}** nevű guild does nem létezik."
+        ],
+        "editedGuild": [
+          "A **{guild}** nevű guild konfigurációja sikeresen frissítve lett."
+        ],
+        "premiumGuild": [
+          "Ez az opció csak a prémium szervereken használható.\n",
+          "Ha többet szeretnél tudni, használd a '/premium információ' parancsot."
```
Comment (plun1331) [inline translations/hu.json]: The whole command name should be translated, including top level
Should also be in backticks for `this effect`

File: translations/hu.json [hunk 22]
```diff
@@ -0,0 +1,2984 @@
...
+      },
+      {
+        "name": "role",
+        "description": "A role ami a ranghoz lesz kötve. Hagyd üresen, ha nem akarsz role-t."
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincs ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a  `/config guilds guild-hozzáadás` paranccsal."
```
Comment (plun1331) [inline translations/hu.json]: The whole command name should be translated, including top level
Should also be in backticks for `this effect`

File: translations/hu.json [hunk 23]
```diff
@@ -0,0 +1,2984 @@
...
+      },
+      {
+        "name": "role",
+        "description": "A role, ami a ranghoz van kötve. Hagyd üresen, ha nem akarsz role-t."
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincs ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a `/config guilds guild-hozzáadás` paranccsal."
```
Comment (plun1331) [inline translations/hu.json]: The whole command name should be translated, including top level
Should also be in backticks for `this effect`

File: translations/hu.json [hunk 24]
```diff
@@ -0,0 +1,2984 @@
...
+      },
+      {
+        "name": "rang",
+        "description": "A törölnivaló rang."
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincs ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a `/config guilds guild-hozzáadása` paranccsal."
```
Comment (plun1331) [inline translations/hu.json]: The whole command name should be translated, including top level
Should also be in backticks for `this effect`

File: translations/hu.json [hunk 25]
```diff
@@ -0,0 +1,2984 @@
...
+      },
+      {
+        "name": "érték",
+        "description": "A követelmény értéke. Írj be akármilyen számot a 'API beállítások bekapcsolva' követelményhez."
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincs ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a `/config guilds guild-hozzáadása` paranccsal."
```
Comment (plun1331) [inline translations/hu.json]: The whole command name should be translated, including top level
Should also be in backticks for `this effect`

File: translations/hu.json [hunk 26]
```diff
@@ -0,0 +1,2984 @@
...
+          "Fairy Souls",
+          "Súlyozás",
+          "API beállítások bekapcsolva"
+        ]
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincs ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a `/config guilds guild-hozzáadása` paranccsal."
```
Comment (plun1331) [inline translations/hu.json]: The whole command name should be translated, including top level
Should also be in backticks for `this effect`

File: translations/hu.json [hunk 27]
```diff
@@ -0,0 +1,2984 @@
...
+      },
+      {
+        "name": "érték",
+        "description": "A követelmény új értéke."
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincs ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a `/config guilds guild-hozzáadása` paranccsal."
```
Comment (plun1331) [inline translations/hu.json]: The whole command name should be translated, including top level
Should also be in backticks for `this effect`

File: translations/hu.json [hunk 28]
```diff
@@ -0,0 +1,2984 @@
...
+      },
+      {
+        "name": "érték",
+        "description": "A követelmény értéke. Írj be akármilyen számot a 'API beállítások bekapcsolva' követelményhez."
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincs ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a `/config guilds add-guild` paranccsal."
```
Comment (plun1331) [inline translations/hu.json]: The whole command name should be translated, including top level
Should also be in backticks for `this effect`

File: translations/hu.json [hunk 29]
```diff
@@ -0,0 +1,2984 @@
...
+      ],
+      "guildNotFound": [
+        "A **{guild}** nevű guild nem létezik."
+      ],
+      "rankNotFound": [
+        "A **{rank}** rang nem létezik a **{guild}** nevű guild-ban."
+      ],
+      "rankNotAdded": [
+        "A **{guild}** nevű guild **{rank}** rangja nincs regisztrálva a bottal.\n",
+        "Regisztráld a `/config guilds add-rank` paranccsal."
```
Comment (plun1331) [inline translations/hu.json]: The whole command name should be translated, including top level
Should also be in backticks for `this effect`

File: translations/hu.json [hunk 30]
```diff
@@ -0,0 +1,2984 @@
...
+      },
+      {
+        "name": "érték",
+        "description": "Az érték, amire a követelményt állítod."
+      }
+    ],
+    "responses": {
+      "guildNotAdded": [
+        "A **{guild}** nevű guild nincsen ehhez a szerverhez kötve.\n",
+        "Add a szerverhez a `/config guilds add-guild` parancshoz."
```
Comment (plun1331) [inline translations/hu.json]: The whole command name should be translated, including top level
Should also be in backticks for `this effect`

File: translations/hu.json [hunk 31]
```diff
@@ -0,0 +1,2984 @@
...
+  },
+  "ping": {
+    "name": "ping",
+    "description": "Megméri a bot lappangási idejét.",
+    "options": [],
+    "responses": {
+      "pong": [
+        "**Pong!**\n",
+        "Pihenő API lappangási idő: {latency:,} ms\n",
+        "Átlagos Websocket lappangási idő : {avg_ws_latency:,} ms\n",
```
Comment (plun1331) [inline translations/hu.json]:
```suggestion
        "Átlagos Websocket lappangási idő: {avg_ws_latency:,} ms\n",
```

File: translations/hu.json [hunk 32]
```diff
@@ -0,0 +1,2984 @@
...
+          "Ajándékok Adva"
+        ]
+      },
+      {
+        "name": "időtartam",
+        "description": "Az esemény időtartama, mindenképp 1 nap és 14 nap között kell lennie. Például: 1w 3d"
+      },
+      {
+        "name": "gvg",
+        "description": "Itt állíthatod be, hogy GvG (Guild versus Guild) legyen az esemény. Ehhez a lehetőséghez prémiummal kell rendelkezni."
```
Comment (plun1331) [inline translations/hu.json]: Guild versus Guild should be translated if possible

File: translations/hu.json [hunk 33]
```diff
@@ -0,0 +1,2984 @@
...
+      {
+        "name": "felhasználó",
+        "description": "A megnézendő felhasználó."
+      }
+    ],
+    "responses": {
+      "notScammer": [
+        "**{player}** nincs megjelölve scammerként/IRL kereskedőként az adatbázisunkban.\n",
+        "Ez ugyanakkor nem jelenti azt, hogy százszázalékosan nem scammel, mindig légy óvatos, amikor más játékosokkal cserélsz.",
+        "\n\n*Ha ez nem igaz, kérjük jelentsd ezt a játékost `/report` paranccsal.*"
```
Comment (plun1331) [inline translations/hu.json]: The command name should be translated
Should also be in backticks for `this effect`

File: translations/hu.json [hunk 34]
```diff
@@ -0,0 +1,2984 @@
...
+      {
+        "name": "játékos",
+        "description": "A megnézendő játékos."
+      }
+    ],
+    "responses": {
+      "notScammer": [
+        "**{player}** nincs megjelölve scammerként/IRL kereskedőként az adatbázisunkban.\n",
+        "Ez ugyanakkor nem jelenti azt, hogy százszázalékosan nem scammel, mindig légy óvatos, amikor más játékosokkal cserélsz.",
+        "\n\n*Ha ez nem igaz, kérjük jelentsd ezt a játékost `/report` paranccsal.*"
```
Comment (plun1331) [inline translations/hu.json]: The command name should be translated
Should also be in backticks for `this effect`

File: translations/hu.json [hunk 35]
```diff
@@ -0,0 +1,2984 @@
...
+  "bits-to-bin": {
+    "name": "bitek-binre",
+    "description": "Kijelzi a bitekért vásárolható tárgyakat, és átváltja coin-ba.",
+    "options": [],
+    "responses": {
+      "title": ["Bit-ek BIN-re"],
+      "field": [
+        "**Bit Price**: {cost:,}\n",
+        "**Coin Price**: {price:,.0f}\n",
+        "**Coins Per Bit**: {coinsPerBit:,.2f}"
```
Comment (plun1331) [inline translations/hu.json:1184]: Translate if possible

File: translations/hu.json [hunk 36]
```diff
@@ -0,0 +1,2984 @@
...
+      }
+    ],
+    "responses": {
+      "notFound": ["Nem találunk információt a **{item}** nevű tárgyról."],
+      "title": ["Tárgy információ: {item}"],
+      "attributesName": ["Attribútomok"],
+      "attributesValue": [
+        "Glowing: {glowing}\n",
+        "Museum Item: {museum}\n",
+        "Dungeon Item: {dungeon}\n"
```
Comment (plun1331) [inline translations/hu.json]: Translate if possible

File: translations/hu.json [hunk 37]
```diff
@@ -0,0 +1,2984 @@
...
+      "entranceTitle": ["{emoji} {name} legjobb menetei az Entrance-on a {profile} profilon"],
+      "floorTitle": ["{emoji} {name} Legjobb menetei a Floor {floor}-on a {profile} profilon"],
+      "masterTitle": ["{emoji} {name} Legjobb menetei a Master Mode Floor {floor}-on a {profile} profilon"],
+      "description": [
+        "Idő: {time}\n",
+        "Eredmény: {score:,}\n",
+        "> Exploration: {exploration_score:,}\n",
+        "> Speed: {speed_score:,}\n",
+        "> Skill: {skill_score:,}\n",
+        "> Bonus: {bonus_score:,}\n",
```
Comment (plun1331) [inline translations/hu.json:2082]: Translate if possible

File: translations/hu.json [hunk 38]
```diff
@@ -0,0 +1,2984 @@
...
+      "description": [
+        "Description: {description}\n",
+        "ID: {id}\n",
+        "Guild Master: {gm}\n",
+        "Members: {members:,}\n",
+        "Created: <t:{created}:F>\n",
+        "Tag: {tag}\n",
+        "XP: {xp:,}\n",
+        "Level: {level:,}\n",
+        "Preferred Games: {prefgames}"
```
Comment (plun1331) [inline translations/hu.json]: Translate if possible

File: translations/hu.json [hunk 39]
```diff
@@ -0,0 +1,2984 @@
...
+      "title": ["A {name} guild tagjai"],
+      "rankInner": [
+        "Members: {members:,}\n",
+        "Created: <t:{created}:F>\n",
+        "Tag: {tag}"
+      ],
+      "memberInner": [
+        "Joined: <t:{joined}:F>\n",
+        "Guild Rank: {rank}\n",
+        "XP History: \n{xp_history}"
```
Comment (plun1331) [inline translations/hu.json]: Translate if possible

File: translations/hu.json [hunk 40]
```diff
@@ -0,0 +1,2984 @@
...
+    "responses": {
+      "title": ["Ellenőrző Instrukciók"],
+      "description": [
+        "Hogy hozzákösd a Minecraft fiókodat a Discord fiókodhoz:\n",
+        "**1)** Akármelyik Hypixel lobbyban jobb-klikkelj a \"My Profile\" tárgyadra a hotbar második helyén.\n",
+        "**2)** Válaszd ki a \"Social Media\" opciót, jobbra a Redstone Blokktól.\n",
+        "**3)** Válaszd ki a \"Discord\" opciót.\n",
+        "**4)** Másold ki a Discord felhasználónevedet ({name}) a chatbe. __Ez kis- és nagybetűérzékeny.__\n",
+        "**5)** Ha egy könyv felbukkan, válaszd a \"I understand\" opciót.\n",
+        "**6)** Használd a `/verify link` parancsot, hogy hozzákösd a Minecraft fiókod a Discord fiókodhoz.\n",
```
Comment (plun1331) [inline translations/hu.json]: The whole command name should be translated, including top level
Should also be in backticks for `this effect`

File: translations/hu.json [hunk 41]
```diff
@@ -0,0 +1,2984 @@
...
+        "note": "Felemelve, amikor egy nem létező kódot akarsz aktiválni.",
+        "message": [
+          "Ez a prémium kód nem érvényes."
+        ]
+      },
+      "15": {
+        "note": "Felemelve, amikor egy eseményt akarsz elindítani, de ezen a szerveren már van elindítva esemény.",
+        "message": [
+          "Már fut egy esemény ezen a szerveren.\n",
+          "Ha már véget ért, kitörölheted a `/event delete` paranccsal."
```
Comment (plun1331) [inline translations/hu.json]: The whole command name should be translated, including top level
Should also be in backticks for `this effect`

File: translations/hu.json [hunk 42]
```diff
@@ -0,0 +1,2984 @@
...
+        ]
+      },
+      "16": {
+        "note": "Felemelve, amikor egy prémium funkciót prémium nélkül akarsz használni.",
+        "message": [
+          "SkyKings Premium szükséges, hogy végrehajtsd ezt a parancsot.\n",
+          "További információért írd be a `/premium info` parancsot."
```
Comment (plun1331) [inline translations/hu.json]: The whole command name should be translated, including top level
Should also be in backticks for `this effect`

File: translations/hu.json [hunk 43]
```diff
@@ -0,0 +1,2984 @@
...
+        "note": "Felemelve, amikor egy esemény nem fut.",
+        "message": [
+          "Nem fut esemény ezen a szereveren."
+        ]
+      },
+      "3": {
+        "note": "Felemelve, amikor a parancsidéző nem ellenőrzött.",
+        "message": [
+          "Ellenőrzöttnek kell lenned, hogy használni tudd ezt a parancsot.\n",
+          "Ellenőrizhetsz a `/verify` paranccsal."
```
Comment (plun1331) [inline translations/hu.json]: The whole command name should be translated, including top level
Should also be in backticks for `this effect`

File: translations/hu.json [hunk 44]
```diff
@@ -0,0 +1,2984 @@
...
+              "**{name}**-nek nincsenek SkyBlock profiljai."
+          ]
+      }
+    },
+    "checkFailure": {
+      "0": {
+        "note": "Felemelve, amikor egy felhasználónak nincs prémiumja, de szükséges.",
+        "message": [
+          "A parancs használatához rendelkezned kell a SkyKings Premium-mal.\n",
+          "Több információért használd a `/premium info` parancsot."
```
Comment (plun1331) [inline translations/hu.json]: The whole command name should be translated, including top level
Should also be in backticks for `this effect`

File: translations/hu.json [hunk 45]
```diff
@@ -0,0 +1,2984 @@
...
+        "note": "Felemelve, amikor a parancsidézőnek nincsenek engedélyei.",
+        "message": [
+          "Nincsenek engedélyeid, hogy használd ezt a parancsot.\n"
+        ]
+      },
+      "8": {
+        "note": "Felemelve, amikor egy prémium funkciót prémium nélkül akarsz használni.",
+        "message": [
+          "SkyKings Premium szükséges, hogy végrehajtsd ezt az akciót.\n",
+          "Több információért használd a `/premium info` parancsot."
```
Comment (plun1331) [inline translations/hu.json]: The whole command name should be translated, including top level
Should also be in backticks for `this effect`

File: translations/hu.json [hunk 46]
```diff
@@ -0,0 +1,2984 @@
...
+      "5": {
+        "note": "Felemelve, amikor a parancsidéző már az eseményben van.",
+        "message": [
+          "Már az eseményben vagy."
+        ]
+      },
+      "6": {
+        "note": "Felemelve, amikor a parancsiddéző nincs az eseményben.",
+        "message": [
+          "Nem vagy az eseményben. Csatlakozz az `/event join` paranccsal."
```
Comment (plun1331) [inline translations/hu.json]: The whole command name should be translated, including top level
Should also be in backticks for `this effect`

**Review by plun1331 — CHANGES_REQUESTED** [review]

File: translations/hu.json [hunk 47]
```diff
@@ -0,0 +1,2984 @@
...
+        "Indok: {reason}"
+      ]
+    }
+  },
+  "report": {
+    "name": "feljelentés",
+    "description": "Jelentesz egy scammer-t, hogy az adatbázishoz adjuk.",
+    "options": [],
+    "responses": {
+      "description": "A scammerek ezen a Google Form-on jelenthetőek: [Google Form](https://forms.gle/ANud6cd9mTx2efbh9)."
```
Comment (plun1331) [inline translations/hu.json]: hello yes i changed original text pls retranslate
https://github.com/SkyKings-Guild/translations/blob/aa6b9f96aa95e9fe07e792ddbe7f6c248048b3c3/translations/en-US.json#L1074

**Comment by plun1331** [comment]
Stale

**Pull request closed without merging**